crates/metal-llvm/src/stmt.rs

```diff
@@ -38,6 +38,8 @@ impl CodeGenValue for FunctionDefinition {
             let entry_block = LLVMAppendBasicBlockInContext(llvm.ctx, function, c"entry".as_ptr());
             LLVMPositionBuilderAtEnd(llvm.builder, entry_block);
 
+            llvm.locals.clear();
```

Comment:
Reason?

Reply:
Clears the locals which may only be present in another function.